crates/holochain_types/src/record.rs

```diff
@@ -365,11 +365,11 @@ impl SignedActionHashedExt for SignedActionHashed {
         let (action, signature) = signed_action.into();
         Self::with_presigned(action.into_hashed(), signature)
     }
-    /// Construct by signing the hash of the action
+    /// Construct by signing the Action (NOT including the hash)
     async fn sign(keystore: &MetaLairClient, action_hashed: ActionHashed) -> LairResult<Self> {
         let signature = action_hashed
             .author()
-            .sign(keystore, action_hashed.as_hash())
+            .sign(keystore, action_hashed.as_content())
```

Comment:
I changed this in the base PR and this essentially reverts it to the original behavior